kubel.el

```diff
@@ -794,6 +806,7 @@ RESET is to be called if the search is nil after the first attempt."
    ("-n" "Tail" "--tail=")]
   ["Actions"
    ("l" "Tail pod logs" kubel-get-pod-logs)
+   ("i" "Tail pod logs (initContainer)" kubel-get-pod-logs--initContainer)
```

Comment:
the phrasing seems a bit awkward. Maybe `Tail initContainer logs`?

Reply:
Thanks! I updated